src/renderer/components/ParentalControlSettings.vue

```diff
@@ -24,6 +24,12 @@
           :default-value="hideSearchBar"
           @change="updateHideSearchBar"
         />
+        <FtToggleSwitch
+          :label="$t('Settings.Parental Control Settings.Hide Uploader')"
```

Comment:
```suggestion
          :label="$t('Settings.Parental Control Settings.Hide Uploader on watch page')"
```

Reply:
Wait do you mean the key should be updated too? Coz the suggested change above does not have it